services/navigation/data/nav_none_cfg.json

```diff
@@ -0,0 +1,34 @@
+{
+    "components":
+    [
+        {
+            "model": "fake",
+            "name": "test_base",
+            "type": "base"
+        }
+    ],
+    "services":
+    [
+        {
+            "name": "test_vision",
```

Comment:
[note] current placeholder vision service, will be updated once explore mode has been built out more. No fake vision service available